### PR TITLE
Capstone Project 

### PR DESCRIPTION
**Pull request by Alondra87**
This final project is based on an online website for a conference. Instead of a conference I created a website for a cultural event.
This link is for the demo of the project: https://www.loom.com/share/90bc35a9c40f489c83f69b6c9133919d

### CODE REVIEW

**Comment by Alondra87** [comment]
Changes made:
>For the mobile version:
I increased the length of the pop-up menu so it covers the page.
 >For the desktop version:
I added the top dark bar like it appears in the template with a section for social media links, logout and language links aligned to the right.
I aligned the logo and text as in the template.
I ensured that the footer on the mobile version has a white background and not a dark one.
I added a border-bottom for the link in program section like in the template.
I added the checked background image behind each speaker, like it is in the template.
I aligned the image and the text in the speaker’s section.
I included a line below every heading as temple.
I aligned images vertically in memories section.

File: index.html [hunk 1]
```diff
@@ -1,13 +1,132 @@
...
+                    <a href="#sponsor" class="nav-link">Sponsors</a> 
+                </li>
+                <li class="nav-item">
+                    <a href="https://alondra87.github.io/Capstone/about.html#memories" class="nav-link">Memories</a> 
+                </li>
+                <li class="nav-item">
+                    <a href="https://www.casadelaculturatarija.com/" class="nav-linkAT">AT&nbsp;Campaign</a> 
+                </li>
+            </ul>
+            <div class="hamburger">
```
Comment (Ismailco) [inline index.html:40]: - [ ] [Optional] The hamburger menu should be aligned to the left of the page, kindly change the menu from the right to the left

|Your design|CC design|
|--|--|
|Screenshot_20220519_210854|Screenshot_20220519_211705|

File: index.html [hunk 2]
```diff
@@ -1,13 +1,132 @@
...
+      </article>
+    </div>
+      <a href="#">SEE THE WHOLE PROGRAM</a>
+      <button type="submit">Join ACF Tarija 2022</button>
+  </section>
+  <section class="featured" id="speakers">
+       <h2>Featured Art Masters</h2><div class="red-line"></div>
+       <div class="art" id="speaker"></div>
+  </section>
+<section class="partners" id="sponsor">
```
Comment (Ismailco) [inline index.html:110]: - [ ] [Optional] The images in this section should be centered with no background or dark background, please consider making the changes so the design is as close to the provided one as possible

|Your design|CC design|
|---|---|
|Screenshot_20220519_212441|Screenshot_20220519_212123|

File: index.html (hunk 1 above)
Comment (Ismailco) [inline index.html:40]: - [ ] [Optional] The nav links should align to the right of the navbar, please consider moving the links to the right and reducing some margin/padding

|Your design|CC design|
|---|---|
|Screenshot_20220519_211814|Screenshot_20220519_211835|

**Review by Ismailco — CHANGES_REQUESTED** [review]
Hi @Alondra87,
Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

giphy

## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by Alondra87** [comment]
I made all the changes, thanks.

**Review by vic778 — APPROVED** [review]
Hi @Alondra87 l  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Charlie Brown

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**